### ISSUE TITLE
Cannot install index.d.ts (type definition file) in 2.2.6 versio

### ISSUE THREAD

**Issue by JaeYeopHan** [JedWatson/classnames#199]
## Description

```sh
$ npm install classnames
```

_in `node_modules`_

![Screen Shot 2019-07-18 at 2 19 41 PM](https://user-images.githubusercontent.com/17924127/61431058-217bd500-a967-11e9-873d-86a88a7362fd.png)

shoul i install `@types/classnames`?

**Comment by rstone770** [comment]
Im having this issue as well and can confirm the are not in included in the package.

**Comment by rstone770** [comment]
I checked the actual tar published on [npm](https://registry.npmjs.org/classnames/-/classnames-2.2.6.tgz) and the types are simply not included. I noticed that there was a [pr]https://github.com/JedWatson/classnames/commit/2c76e87c4afdc7f6ceec5e10c0b9dba08cb4dcbb#diff-b52768974e6bc0faccb7d4b75b162c99) done about a year ago, maybe the package was published with an older version of NPM that did not contain implicit type entry support and was simply ignore since .npmignore ignores everything.  Although i find it suspicious that only 2 users have reported this so far.

ive installed @types/classnames as a stop gap to stop ts from complaining.
